### PR TITLE
Inner rotor

### PR DESCRIPTION
**Pull request by Nick-Hemenway**

### CODE REVIEW

**Comment by npetersen2** [comment]
Seems very good. Only concern I have is code readability. In cross section classes, code which calculates points and transforms them is a bit hard to follow:

From trapezoid cross section:
```MATLAB
    x = [-(w/2), -(w/2) + (h/tan(theta)), ...
          (w/2) - (h/tan(theta)), (w/2)];
            
    y = [0, h, h, 0];
            
    [x_trans, y_trans] = obj.location.transformCoords(x,y);

    point1 = [x_trans(1), y_trans(1)];
    point2 = [x_trans(2), y_trans(2)];
    point3 = [x_trans(3), y_trans(3)];
    point4 = [x_trans(4), y_trans(4)];
```
You completely separate the x from y coordinates. This is a little hard to understand when reading the code.

For example, if I want to know what the equation is for `point3`, I have to go find `x_trans` and `y_trans` separately, then parse out the matrix line to find: `point3 = [(w/2) - (h/tan(theta)), h]`. Maybe we can directly write that question, then transform on the whole `[x,y]` object? Storing `x` and `y` separately for each point seems unclear.

(This style is also done in other cross section classes)

**Comment by npetersen2** [comment]
Another minor comment:
In `CrossSectInnerRotorPMStator.m`, when you validate the attributes, isn't `stator_slots` an integer? So following code should not validate `double`, but maybe `int` or equivalent?

```c
validateattributes(obj.stator_slots,{'double'},{'nonnegative','nonempty'})
```

**Comment by elsevers** [comment]
- Missing method documentation
- When I run the example I get a lot of these messages (see below). Do we need to add more terminating `;`?
```
ans =

   NaN
```

**Comment by elsevers** [comment]
I have created a pull request into this branch to change `stator_slots` to `Q`.

**Comment by elsevers** [comment]
Please split into two pull requests:

1. a pull request we can complete right now (after you resolve merge conflicts) that does not completely implement fillets.
2. a pull request that adds the fillets.

**Comment by elsevers** [comment]
`CrossSectInnerRotorPMStator.m` should be `CrossSectInnerRotorStator.m` (does not have anything to do with whether the rotor has PMs)

**Comment by npetersen2** [comment]
The example works as is, but doesn't work to extrude multiple cross sections. (i.e., changing the `cs` variable to include all `stator`s). We can just merge the code as is.

**Review by npetersen2 — DISMISSED** [review]
The example works as is, but doesn't work to extrude multiple cross sections. (i.e., changing the `cs` variable to include all `stator`s). We can just merge the code as is.

**Review by elsevers — APPROVED** [review]
Looks good. @Nick-Hemenway, please continue the improvements you mention in a new PR.

**Review by npetersen2 — APPROVED** [review]
